merge_request_approvals_test.go

```diff
@@ -188,6 +188,7 @@ func TestGetApprovalRules(t *testing.T) {
 				"id": 1,
 				"name": "security",
 				"rule_type": "regular",
+				"section": "test",
```

Comment:
Please place it in the "correct" location, looking at the output that should be just below `contains_hidden_groups`. If you can do that for all instances that would be great.

Reply:
I restructured it